model/src/main/java/com/javahelp/model/token/Token.java

```diff
@@ -0,0 +1,123 @@
+package com.javahelp.model.token;
+
+import com.javahelp.model.user.User;
+import org.joda.time.DateTime;
+import java.security.SecureRandom;
+
+/**
+ * Representation of token entity
+ */
+
+public class Token {
+    /**
+     * Token requested by the user
+     */
+    private String token;
+
+    /**
+     * Unique id of the token.
+     */
+    private String id;
```

Comment:
We may want to get rid of the id field, and simply leave the token field.

Reply:
The argument for this is that the id is somewhat redundant, as the token string is already created at the same time as the token is, and is already immutable. We can just use the token string as the id.

Reply:
You are right. I forgot to remove the id.